src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -915,6 +925,39 @@ const renderValidationErrorLabel = (message?: string): JSX.Element => {
     )
 }
 
+export const ValueNameInput = ({
+    valueName,
+    handleValueNameChange,
+    handleValueNameOnBlur,
+    invalidValueName,
+    invalidValueNameMessage,
+    valueNameDisabled,
+}: {
```

Comment:
create a type and move to ChartValuesView.type

Reply:
Done